client/src/app/core/repositories/motions/motion-repository.service.ts

```diff
@@ -175,12 +175,29 @@ export class MotionRepositoryService extends BaseIsAgendaItemAndListOfSpeakersCo
         return { title: this.getAgendaSlideTitle(viewMotion), subtitle };
     };
 
-    protected createViewModelWithTitles(model: Motion): ViewMotion {
+    protected createViewModel(model: Motion): ViewMotion {
         const viewModel = super.createViewModel(model);
 
         viewModel.getNumberOrTitle = () => this.getNumberOrTitle(viewModel);
         viewModel.getProjectorTitle = () => this.getProjectorTitle(viewModel);
 
+        viewModel.getAmendmentParagraphLines = () => {
+            if (viewModel.lead_motion && viewModel.isParagraphBasedAmendment()) {
```

Comment:
This code was originally a "relation": https://github.com/OpenSlides/OpenSlides/blob/master/client/src/app/core/repositories/motions/motion-repository.service.ts#L326

Since I redone the relations I deleted it and as I just saw I never readded the code again. Here it is ;)